scipy/stats/__init__.py

```diff
@@ -413,6 +413,14 @@
    circvar
    circstd
 
+Directional statistical functions on the unit sphere
+----------------------------------------------------
+
+.. autosummary::
+   :toctree: generated/
+
+   directionalmean
+
```

Comment:
What do you think about having this and "Circular Statistics" under the more general banner of "Directional Statistics" until we have more functions? Would that be accurate?

Reply:
Yep, would make sense in my opinion too.